### PR TITLE
Fix python static position

### PR DESCRIPTION
**Pull request by nbuffon**
Fix GPS time position management in python client

### CODE REVIEW

**Comment by nbuffon** [comment]
@tigroo FYI

**Comment by ymorin-orange** [comment]
Fix for `--geo-position=static` confirmed.

File: python/its_client/mqtt/mqtt_worker.py [hunk 1]
```diff
@@ -68,10 +69,22 @@ def step(self) -> bool:
...
+                        logging.warning(
+                            f"the position time is {difference.seconds * 1000 + difference.microseconds / 1000} ms in the past"
+                        )
+                else:
+                    difference = gps_utc_datetime - utc_now
+                    if difference.microseconds > 1000:
+                        logging.warning(
+                            f"the position time is {difference.seconds * 1000 + difference.microseconds / 1000} ms in the future"
+                        )
+
```
[the remark this answers is not in the document]
Comment (nbuffon) [reply python/its_client/mqtt/mqtt_worker.py:87]: Shorter, but simpler ? :hand_over_mouth: 
Will do the update :)

[the remark this answers is not in the document]
Comment (ymorin-orange) [reply python/its_client/mqtt/mqtt_worker.py:80]: It's both simpler and shorter, no? ;-p

**Review by ymorin-orange — APPROVED** [review]

**Pull request merged**